README.md

```diff
@@ -31,34 +31,50 @@ critical things like tuning performance and getting the most out of
 Spanner. Expect that you'll need to tweak and enhance what HarbourBridge
 produces.
 
-To use the tool on a PostgreSQL database called mydb, run
+Below are some quick starter examples on how to run HarbourBridge. You can look
+at specific sections for PostgreSQL, MySQL etc for detailed examples.
+
+For example, to generate Spanner schema from a PostgreSQL dump file, run
 
 ```sh
-# By default, the driver is "pg_dump".
-pg_dump mydb | harbourbridge
-# Or,
-pg_dump mydb | harbourbridge -driver=pg_dump
+# Generate pg_dump file.
```

Comment:
This generates a fairly intimidating set of instructions! And it's not clear as a first time user what I should do since many of the instructions are alternatives: https://screenshot.googleplex.com/m3vBNwSeKJirNSL

I would keep the first set of instructions super-simple. I think the simpler the better.
How about just:

# Generate pg_dump file.
pg_dump mydb > mydb.pg_dump
# Run HarbourBridge
harbourbridge schema -source=postgresql < mydb.pg_dump

And then have a separate block of instructions for a GCS bucket. Maybe something like:

You can also specify path to a file stored in GCS (Google Cloud Storage) via:

harbourbridge schema -source=postgresql -source-profile="format=dump,file=gs://<bucket_name>/<file_path>"

And just leave it at that. 
I would drop the other options -- this is the intro part of the README, and I don't think we need to be complete here.

Reply:
I have now dropped the examples from the Intro section altogether and we simply ask users to look at the quick start guide. This takes care of the duplication as well as moves documentation about GCS to where we describe `file` param in `-source-profile` flag.